src/mappings/gns.ts

```diff
@@ -197,15 +197,16 @@ export function handleSubgraphPublished(event: SubgraphPublished): void {
   let subgraphNumber = event.params.subgraphNumber.toString()
   let subgraphID = joinID([graphAccountID, subgraphNumber])
   let versionID: string
-  let versionNumber: number
+  let versionNumber: BigInt
 
   // Update subgraph
   let subgraph = createOrLoadSubgraph(subgraphID, event.params.graphAccount, event.block.timestamp)
   let oldVersionID = subgraph.currentVersion
 
   versionID = joinID([subgraphID, subgraph.versionCount.toString()])
   subgraph.currentVersion = versionID
-  subgraph.versionCount = subgraph.versionCount.plus(BigInt.fromI32(1))
+  versionNumber = subgraph.versionCount.plus(BigInt.fromI32(1))
+  subgraph.versionCount = versionNumber
```

Comment:
Isn't this incorrect? `versionNumber` should be the value of `subgraph.versionCount` before incrementing it (since it's what we are using to create the ID which is used in line `221` to create the entity.

Reply:
It works either way I think, but your suggestion would make it more consistent (count from 0 instead of from 1.) I'll update.